src/components/message/message.component.tsx

```diff
@@ -39,52 +39,82 @@ export interface MessageProps extends MarginProps {
   variant?: MessageVariant;
 }
 
-export const Message = ({
-  open = true,
-  transparent = false,
-  title,
-  variant = "info",
-  children,
-  onDismiss,
-  id,
-  className,
-  closeButtonAriaLabel,
-  showCloseIcon = true,
-  ...props
-}: MessageProps) => {
-  const marginProps = filterStyledSystemMarginProps(props);
-  const l = useLocale();
-  const renderCloseIcon = () => {
-    if (!showCloseIcon || !onDismiss) return null;
+export const Message = React.forwardRef<HTMLDivElement, MessageProps>(
+  (
+    {
+      open = true,
+      transparent = false,
+      title,
+      variant = "info",
+      children,
+      onDismiss,
+      id,
+      className,
+      closeButtonAriaLabel,
+      showCloseIcon = true,
+      ...props
+    }: MessageProps,
+    ref
```

Comment:
It might be worth documenting `ref` in storybook so consumers are aware it is available for use for them.